e2e/ls_test.go

```diff
@@ -786,3 +786,19 @@ func TestListNestedLocalFolders(t *testing.T) {
 		2: match(filepath.ToSlash("file.txt")),
 	}, trimMatch(dateRe), alignment(true))
 }
+
+func TestEmptyBucket(t *testing.T) {
+	t.Parallel()
+
+	s3client, s5cmd := setup(t)
+
+	bucket := s3BucketFromTestName(t)
+	createBucket(t, s3client, bucket)
+
+	cmd := s5cmd("ls", "s3://"+bucket)
+	result := icmd.RunCmd(cmd)
+
+	result.Assert(t, icmd.Success)
+
+	assertLines(t, result.Stdout(), nil)
+}
```

Comment:
This testcase is unrelated to this PR.

Reply:
This PR contains changes from [737](https://github.com/peak/s5cmd/pull/732). I adjusted test cases according to the scenario where both PRs merged to the master branch. Other PRs changes must be present for the CI to pass in this branch. 
Therefore, this PR can be merged after [737](https://github.com/peak/s5cmd/pull/732).